packages/api-v2/test/student/student.e2e-spec.ts

```diff
@@ -26,12 +29,10 @@ describe("StudentController (e2e)", () => {
     uuid = res.body.uuid;
   });
 
-  afterEach(async () => {
-    await dropStudentTable(connection);
-  });
-
   afterAll(async () => {
```

Comment:
Hmm I'm not sure about having tests depend on each other. I'm assuming if we don't reset the user table after each test, we are persisting the environment throughout each test. Is it not better practice to reset the env before each test?

Reply:
True that's better practice, but it doesn't work the other way and I don't know why lol

Reply:
Someone else in 3 semesters can figure it out

Reply:
BRUH LMAO ok i make a ticket